docs/pages/get-started.mdx

```diff
@@ -44,27 +44,28 @@ sessions, so you can review them later.
   in Teleport without using Docker, see the getting started guide for
   [server access](enroll-resources/server-access/getting-started.mdx).
 
-- The `tsh` client tool. 
+- The `tsh` client tool. `tsh` is included in the same packages and archives as
+  the `teleport` binary. See the [Installation Guide](installation.mdx) for how
+  to install Teleport on your operating system.
   
-  The `tsh` client tool is only required to access the server from a terminal.
-  For installation instructions, see [Installation Guide](installation.mdx).
-  If you don't install the `tsh` client tool, you can access the server in
-  Teleport Enterprise (Cloud) using the Web UI through your browser.
+  This guide only requires the `tsh` client tool to access a server from a
+  terminal. If you don't install the `tsh` client tool, you can access the
+  server in Teleport Enterprise (Cloud) using the Web UI through your browser.
 
 ## Step 1/5. Spin up a server
 
 To spin up a new server using Docker:
 
-1. Open a terminal shell on your workstation.
+1. Open a terminal on your workstation.
 
 1. Start a Docker container on your workstation to prepare a server that you want enroll 
 as a resource in your Teleport Enterprise (Cloud) cluster:
    
    ```code
-   $ docker run --interactive --tty ubuntu:22.04 /bin/bash
+   $ docker run --interactive --tty ubuntu:24.04 /bin/bash
```

Comment:
LGTM assuming we verified that this guide still works with Ubuntu 24.04.

IIRC some of our install scripts or systemd units struggle with Ubuntu 24.04..